tests/unit/test-wcs-compatibility-functions.php

```diff
@@ -23,12 +23,18 @@ public function test_wcs_get_objects_property() {
 		$this->assertEquals( get_post( $simple_product->get_id() ), wcs_get_objects_property( $simple_product, 'post' ) );
 		$this->assertEquals( get_post( $variation_product->get_parent_id() ), wcs_get_objects_property( $variation_product, 'post' ) );
 
-		// post_status.
+		// post_status - now deprecated as a property
```

Comment:
If these are deprecated can we remove them from the unit tests?
Or move to another function and mark as deprecated?

Reply:
This param isn't deprecated (copy/paste fail from original PR made to WC Subscriptions). I've removed this wording from the comment to avoid confusion: [4c72ed4](https://github.com/Automattic/woocommerce-subscriptions-core/pull/159/commits/4c72ed41d623c4ed5f43f3980e845b24b07f0f6e)